### PR TITLE
Feature 6156040636 webpage updates

### PR DESCRIPTION
**Pull request by RyanCrux**
## What type of PR is this? (check all applicable)

- [x] Feature
- [ ] Bug Fix
- [ ] Hot Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Multiple edits requested across three pages. Provided list can be found uploaded in "files" on the Monday ticket:
https://blackberg-group.monday.com/boards/1931719392/pulses/6156040636

Overview:

Homepage:
- Update Diffusion Talent Accelerator Innovation Spotlight with new image
- Update News & Events Carousel ( removed old and replaced [image/text/links] )
- Update Our Team Section (multiple new/updated head shots) 

shark tank page:
- Replace innovation spotlight for Surgical Pause with Contraception on Demand (image/text/links)
- Replace innovation spotlight for Beta Lactum with White Noise Project (image/text/links)
- Update News & Events Carousel ( removed old and replaced [image/text/links] )

National Diffusion Practices page:
- Update News & Events Carousel ( removed old and replaced [image/text/links] )

One remaining request about overall font size on site was handed off to Will. This was removed from the scope of this ticket at this time.

## Related Tickets & Documents
- Monday Task #6156040636

## QA Instructions, Screenshots, Recordings

review new images and check page paths. 

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
- [ ] Semantic HTML implemented?
- [ ] Keyboard operability supported?
- [ ] Checked with [Google Lighthouse](https://developer.chrome.com/docs/lighthouse/overview) and addressed `Critical` and `Serious` issues?
- [ ] Color contrast tested?

## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _site/assets/img/sharktank/contraception-on-demand.jpg
 - _site/assets/img/sharktank/white-noise-project.jpg
 - assets/img/diffusion-practices/card4.jpg
 - assets/img/diffusion-practices/card5.jpg
 - assets/img/diffusion-practices/card6.jpg
 - assets/img/diffusion-practices/card7.jpg
 - assets/img/home/news_8.jpg
 - assets/img/home/news_9.jpg
 - assets/img/sharktank/contraception-on-demand.jpg
 - assets/img/sharktank/research.jpg
 - assets/img/sharktank/sharktank-finals-revealed.jpg
 - assets/img/sharktank/st.jpg
 - assets/img/sharktank/vha-sharktank.jpg
 - assets/img/sharktank/white-noise-project.jpg
 - _includes/home/doe-team.html
 - _includes/sharktank/3-up.html
 - _site/sharktank/index.html
 - assets/img/home/row5.jpg
 - assets/img/home/team-fitzgerald.png
 - assets/img/home/team-goetschius.png
 - assets/img/home/team-hagan.png
 - assets/img/home/team-henderson.png
 - assets/img/home/team-mccoy.png
 - assets/js/doe-news.js
 - assets/js/practices-news.js
 - assets/js/sharktank-news.js

### CODE REVIEW

**Comment by RyanCrux** [comment]
Compressed images and all checks have passed - I need one reviewer please :)

File: _includes/sharktank/3-up.html [hunk 1]
```diff
@@ -27,15 +27,17 @@
                 </svg>                    
             </div>
             <div class="three-stories__content tablet-lg:grid-col-6">
-                <h2 class="margin-bottom-1">The Surgical Pause</h2>
-                <h3 class="margin-y-0">Reducing Post-surgery Mortality in Frail Veterans</h3>
-                <p class="font-body">The Surgical Pause utilizes the simple yet sophisticated Risk Analysis Index (RAI) to screen Veterans for frailty in 30 seconds, effectively flagging high-risk Veterans so that the surgical team can ensure that the proposed treatment plans both mitigate known risks and align with the Veterans' overarching life goals.</p>
-                <a href="https://marketplace.va.gov/innovations/preoperative-frailty-screening-prehabilitation" class="usa-button button-vha">More Information</a>
+                <h2 class="margin-bottom-1">Contraception on Demand </h2>
+                <h3 class="margin-y-0">Expanding access to contraceptive care for Veterans</h3>
+                <p class="font-body">o	Contraception on Demand trains pharmacy practitioners to conduct patient-centered
```
Comment (will-bbg) [inline _includes/sharktank/3-up.html]: Weird character in the first line of the sentence.

**Review by will-bbg — DISMISSED** [review]
Team photos are not updated
Screenshot 2024-03-07 at 8 15 30 AM

- News and Events are not updated
Screenshot 2024-03-07 at 8 20 15 AM

Please double check that the files were uploaded to Dev-Innovation. This is also a part of the PR creation process

**Comment by RyanCrux** [comment]
@will-bbg fixed up the weird character and adjusted my paths correctly. I verified everything on dev-innovation.bgsandbox. Please take another pass at approving, thanks!

File: _includes/sharktank/3-up.html [hunk 2]
```diff
@@ -65,15 +67,16 @@ <h3 class="margin-y-0">Veterans Mental Evaluation Team</h3>
...
+                  to hospitals across the country, providing patients with increased sleep and decreased anxiety in the hospital and at home.</p>
+                <a href="https://marketplace.va.gov/innovations/the-white-noise-project" class="usa-button button-vha">More Information</a>
             </div>
             <div class="tablet-lg:grid-col-5 tablet-lg:grid-offset-1 desktop-lg:grid-col-5 desktop-lg:grid-offset-1 widescreen:grid-col-5 widescreen:grid-offset-1 mobile:padding-top-5 desktop:padding-top-0">
                 <div class="three-stories__img">
                     <div class="three-stories__img-container">
-                        <a href="https://marketplace.va.gov/innovations/beta-lactam-allergy-assessment-saving-lives-one-assessment-at-a-time"><img src="../assets/img/sharktank/beta-lactam-allergy-assessment.jpg" class="img-fluid radius-lg" alt="Poster image of contestants and judges on Shark Tank Competition stage" /></a>
+                        <a href="https://marketplace.va.gov/innovations/the-white-noise-project"><img src="../assets/img/sharktank/white-noise-project.jpg" class="img-fluid radius-lg" alt="Image of person presenting on stage" /></a>
```
Comment (ronald-bbg) [inline _includes/sharktank/3-up.html:79]: For alt text best practices, I would remove "Image of" and say "A person presenting..."

File: _includes/sharktank/3-up.html [hunk 3]
```diff
@@ -27,15 +27,17 @@
...
+                  experiences with reproductive care at VA and expanding the scope of practice for VA clinical pharmacists.</p>
+                <a href="https://marketplace.va.gov/innovations/contraception-on-demand" class="usa-button button-vha">More Information</a>
             </div>
             <div class="tablet-lg:grid-col-5 tablet-lg:grid-offset-1 desktop-lg:grid-col-5 desktop-lg:grid-offset-1 widescreen:grid-col-5 widescreen:grid-offset-1 mobile:padding-top-5 desktop:padding-top-0">
                 <div class="three-stories__img">
                     <div class="three-stories__img-container">
-                        <a href="https://marketplace.va.gov/innovations/preoperative-frailty-screening-prehabilitation"><img src="../assets/img/sharktank/surgical-pause.jpg" class="img-fluid radius-lg" alt="Poster image of contestants and judges on Shark Tank Competition stage" /></a>
+                        <a href="https://marketplace.va.gov/innovations/contraception-on-demand"><img src="../assets/img/sharktank/contraception-on-demand.jpg" class="img-fluid radius-lg" alt="Image of people sitting around a table" /></a>
```
Comment (ronald-bbg) [inline _includes/sharktank/3-up.html:40]: Can remove "Image of" and just describe who is in the photo and what they are doing

**Review by ronald-bbg — CHANGES_REQUESTED** [review]
Left a couple notes on alt text best practices where we can remove "Image of" and directly describe the person doing an action...for example "A person conversating with..."

**Review by ronald-bbg — APPROVED** [review]

**Pull request merged**